src/domUpdates.js

```diff
@@ -23,7 +23,35 @@ let domUpdates = {
   },
 
   populateGameBoard(currentBoard){
```

Comment:
This is the code for our board, Any suggestions on how to refactor? @robbiejaeger

Reply:
Loops! Try using some good old-fashioned for loops, or doing a `forEach` over your `currentBoard` array.